src/UI/Component/Item/Item.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Item;
+
+/**
+ * Common interface to all items.
+ */
+interface Item extends \ILIAS\UI\Component\Component {
+	/**
+	 * Gets the title of the item 
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Create a new item with an attached description.
+	 * @param string $description
+	 * @return AppointmentItem
+	 */
+	public function withDescription($description);
+
+	/**
+	 * Get the description of the item.
+	 * @return string
+	 */
+	public function getDescription();
+
+	/**
+	 * Get a new item with the given properties as key-value pairs.
+	 *
+	 * The key is holding the title and the value is holding the content of the
+     * specific data set.
+	 *
+	 * @param array<string,string> $properties Title => Content
```

Comment:
No, we are not sure. I kept this from the original proposal. I think a key/value array will not do this in the long run. E.g. if properties should be linked (like in the appointment list mockup). If we have a general Link component, I will extend/revise this part. My current hope was that accepted stuff could stay "as is" for now, see https://github.com/ILIAS-eLearning/ILIAS/pull/399#issuecomment-271278679

Label instead of title would be ok for me though.